packages/site/src/content/DataTable/DataTableNotes.mdx

```diff
@@ -0,0 +1,183 @@
+  </DataTable.Cell>
+  <DataTable.RowActions>
+    <Button icon="edit" label="Edit" />
+    <Button icon="delete" label="Delete" />
+  </DataTable.RowActions>
+</DataTable.Row>
+```
+
+### Sortable Headers
+
+For sortable columns, use DataTable.SortableHeader with the required props:
+
+```tsx
+<DataTable.Header>
+  <DataTable.SortableHeader
+    direction={
+      currentSort === "desc"
+        ? SortDirection.descending
+        : currentSort === "asc"
+        ? SortDirection.ascending
+        : SortDirection.equilibrium
+    }
+    onSort={() => handleColumnSort("name")}
+  >
+    Name
+  </DataTable.SortableHeader>
+  <DataTable.HeaderCell>Email</DataTable.HeaderCell>
+</DataTable.Header>
+```
+
+The `direction` prop accepts:
+
+- `SortDirection.ascending` - ascending sort
+- `SortDirection.descending` - descending sort
+- `SortDirection.equilibrium` - no sort applied
+
+When both `direction` and `onSort` are provided, the header becomes interactive
+with a sort icon. When either is missing, it renders as a regular header cell.
+
+The `direction` prop represents the current sort state of the column. Your
+`onSort` callback should handle the state transitions (ascending → descending →
+equilibrium) based on your application's logic.
+
+### Pagination
+
+For pagination functionality, use DataTable.Pagination and
+DataTable.PaginationButton:
+
+```tsx
+<DataTable.Footer>
+  <DataTable.Pagination>
+    <DataTable.PaginationButton
+      direction="previous"
+      disabled={!canGoPrevious}
+      onClick={handlePreviousPage}
+      ariaLabel={direction =>
+        direction === "next" ? "Next page" : "Previous page"
+      }
+    />
+    <DataTable.PaginationButton
+      direction="next"
+      disabled={!canGoNext}
+      onClick={handleNextPage}
+      ariaLabel={direction =>
+        direction === "next" ? "Next page" : "Previous page"
+      }
+    />
+  </DataTable.Pagination>
+</DataTable.Footer>
+```
+
+**Important**: The `ariaLabel` prop is required and should be a function that
+takes a direction parameter and returns translated strings for accessibility.
+Consider using your application's translation system:
+
+```tsx
+ariaLabel={(direction) =>
+  direction === "next" ? t("Next page") : t("Previous page")
+}
+```
+
+### Setting Up Your Own Provider
+
+Atomic DataTable components are fully functional on their own, no provider is
+required. You can render rows, cells, headers, and actions manually, using
+static data or your own logic.
+
+However, if you're integrating with a table library like TanStack Table, you may
+choose to create a context-based provider to pass the table instance down to the
+child components. This allows you to centralize state management, such as
+sorting, pagination, and row models, while keeping your table layout clean and
+declarative.
+
+This pattern is entirely optional, but it can help streamline more complex or
+dynamic table setups.
+
+{/* TODO: Link to examples in storybook once in production */}
```

Comment:
Leaving this here on purpose - I want to link to some examples in Storybook but can't do that until the stories are in prod

Reply:
It's more that I knew they wouldn't work in the branch before being on prod but you're right - we can just link them and once it merges it'll work. Done here 064da5ba8ed4bd679a8c27f4f722188031520547